decidim-core/app/packs/src/decidim/input_emoji.js

```diff
@@ -26,6 +26,23 @@ export class EmojiButton {
     return I18N_CONFIG;
   }
 
+  // Get the current locale used for the emoji database
+  //
+  // @see {@link https://emojibase.dev/docs/datasets/#supported-locales|Supported locales in the Emojibase documentation site}
+  // @see {@link https://github.com/milesj/emojibase/blob/master/packages/core/src/constants.ts}|the SUPPORTED_LOCALES constant}
+  //
+  // @returns {string} the current locale if it's supported by emoji base, or english as the fallback locale
+  static locale() {
+    const emojibaseSupportedLocales = ["da", "de", "en", "en-gb", "es", "es-mx", "et", "fi", "fr", "hu", "it", "ja", "ko", "lt", "ms", "nb", "nl", "pl", "pt", "ru", "sv", "th", "uk", "zh", "zh-hant"];
```

Comment:
You can get the supported locales with the following code:

```js
import { SUPPORTED_LOCALES } from "emojibase";

let emojiLocale = document.documentElement.getAttribute("lang");
if (!SUPPORTED_LOCALES.includes(emojiLocale)) {
  emojiLocale = "en"
}
```

And maybe we could also look for a secondary alternative using the first part of the locale:

```js
if (!SUPPORTED_LOCALES.includes(emojiLocale)) {
  const secondaryLocale = emojiLocale.split("-")[0];
  if (SUPPORTED_LOCALES.includes(secondaryLocale)) {
    emojiLocale = secondaryLocale;
  } else {
    emojiLocale = "en";
  }
}
```

Reply:
Oh, of course! I didn't realize that this was an exported constant. Much better, of course!! I'll make the changes in a couple of hours.

Reply:
Done in b6b6b7d7fc64dabcde5703ee6f55ad8841575dd2 
I've also added a spec. 
This is ready to another review